tests/asgi/tests.py

```diff
@@ -333,10 +335,11 @@ def __call__(self, **kwargs):
         response_body = await communicator.receive_output()
         self.assertEqual(response_body["type"], "http.response.body")
         self.assertEqual(response_body["body"], b"Hello World!")
+
         # Give response.close() time to finish.
         await communicator.wait()
 
-        # AsyncToSync should have executed the signals in the same thread.
+        # SyncToAsync should have executed the signals in the same thread.
```

Comment:
If there is a typo here, it's unrelated with this patch. Please send a fix separately.

Reply:
ok yeah, I'm pretty sure there is (and I had previously modified this test, but I'll revert all the changes)

Reply:
sure (I think there is a typo, and in a previous revision I'd made substantial modifications here, but I'll put it back for now)